internal/assets/skills/issue-creation/references/delegated-workflow-actions.md

```
@@ -0,0 +1,40 @@
+# Delegated Workflow Actions
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)